README.rst

```diff
@@ -668,6 +668,7 @@ All available options:
                                        python-<debianized-setup-name>)
   Suite                                suite (e.g. stable, lucid) in changelog
                                        (Default: unstable)
+  Suite3                               suite (e.g. stable, lucid) for python3
```

Comment:
Nitpick: how about matching the doc of `Suite` and other Python 3 options more closely:

```
  Suite2                               suite (e.g. stable, lucid) in changelog
                                       for python 3 (Default: unstable)
```

Reply:
2821f4d updates this. Since Suite3 doesn't have an independent default I used the phrase `Default: uses value of Suite option.`